src/pveclib/vec_int128_ppc.h

```diff
@@ -1957,7 +1957,7 @@ example_longdiv_10e31 (vui128_t *q, vui128_t *d, long int _N)
  */
 #define CONST_VUINT128_QxD(__q0, __q1) ( (vui128_t) \
     (((unsigned __int128) __q0) << 64) \
-    + (((unsigned __int128) __q1) )
+    + ((unsigned __int128) __q1) )
```

Comment:
tucked bugfix?

Reply:
Yup, found the extra "(" while working on testprograms